nautobot/utilities/tables.py

```diff
@@ -60,6 +76,12 @@ def __init__(self, *args, user=None, **kwargs):
 
         # Dynamically update the table's QuerySet to ensure related fields are pre-fetched
         if isinstance(self.data, TableQuerysetData):
+            # Extract custom field values
+            cf_fields = {}
+            for key, cf in custom_fields.items():
+                cf_fields[key] = Func(F("_custom_field_data"), Value(cf.name), function="jsonb_extract_path_text")
```

Comment:
This line is causing a crash on MySQL (See: #524). I was testing your branch, and it took me a minute to realize it was this line here. We're going to have to think a little more about how to solve this agnostically for both PG and MySQL, seeing as we're officially launching MySQL support in v1.1.0.

Reply:
Django supports lookups that descend into JSON documents: https://docs.djangoproject.com/en/3.2/topics/db/queries/#key-index-and-path-transforms